.github/workflows/build.yml

```diff
@@ -1,13 +1,10 @@
 ---
-name: build
-
+name: Lint and test
 on:
   pull_request:
     branches: [main]
-
 env:
-  PYTHON_VERSION: '3.9'
-
+  PYTHON_VERSION: '3.11'
```

Comment:
In the `pyproject.toml` we have a lower bound for 3.9, we need to test for this as well then, so we do not introduce syntax which is not allowed in 3.9.

Reply:
Ok